configuration/default-applications.md

```diff
@@ -0,0 +1,107 @@
+
+Some default applications can be set (system-wide) with the [Debian Alternatives System](https://wiki.debian.org/DebianAlternatives), noticeably the editor, terminal emulator and (sometimes) the web browser. You do this by running `sudo update-alternatives --config name` with `name` being e.g. `editor`, `x-terminal-alternatives`, (sometimes) `x-www-browser`, `vi`, etc. Other things you can set the default applications for are listed in the directory `/etc/alternatives/`. See `man update-alternatives` for more information about the command.
+
+Since the changes are not saved in the home directory, these changes must be set in a TemplateVM, or they will be reset when the AppVM ist shut down. (Confused? See [Get Started](https://www.qubes-os.org/getting-started/).) All of the following settings are saved to the home directory.
+
+You can also set the default application according to [media (or MIME) types](https://en.wikipedia.org/wiki/Media_type).
+
+Run `mimetype filename` to get the MIME type of that file type. Example:
+
+	$ mimetype some_document.pdf
+	some_document.pdf: application/pdf
+
+Run `xdg-mime query default mime-type` to see which application opens this MIME type by default. Example:
+
+	$ xdg-mime query default application/pdf
+	evince.desktop
+
+To open a file in its default application run `xdg-open filename`.
+
+**Note:** If `xdg-mime` and/or `xdg-open` is not working as expected [you might need](https://askubuntu.com/questions/779717/xdg-open-stopped-working-since-16-04-upgrade) to install `qvfs-bin`. (This has happend in a debian-8 template updated to debian-unstable. It does not seem to happen in debian-minimal templates.)
+
+To set a new default application for a filetype run `xdg-mime default application.desktop mime-type`. `application.desktop` has to be a [Desktop entry](https://wiki.archlinux.org/index.php/Desktop_entries) in `/usr/source/applications/` or `~/.local/share/applications`. Example:
+
+	$ xdg-mime default zathura-pdf-poppler.desktop application/pdf
+
+`xdg-mime` confirms:
+
+	$ xdg-mime query default application/pdf
+	zathura-pdf-poppler.desktop
+
+This setting is usually effective immediately and is written to `~/.local/share/mimeapps.list`, i.e.:
+
+	[Default Applications]
+	application/pdf=zathura-pdf-poppler.pdf
+
+See below for a more extensive `mimeapps.list` file.
+
+(If you want to apply this setting for every AppVM of a TemplateVM you need to copy `mimeapps.list` to `/usr/share/applications/` in the TemplateVM)
+
+
+## Setting `qvm-open-in-(d)vm` as a default application for file types/URLs
+
+Desktop entry files contain, in essence, instructions for executing a command. This turns out to be quite useful in combination with Qubes' [`qvm-open-in-dvm`](https://www.qubes-os.org/doc/vm-tools/qvm-open-in-vm/) and [`qvm-open-in-dvm`](https://www.qubes-os.org/doc/vm-tools/qvm-open-in-dvm/), which open a specified file (or URL) in a new DispVM or in a specific VM, respectively.
```

Comment:
The first reference should be `qvm-open-in-vm` (the link is fine, only label is wrong).
And the end of sentence should have the same order of `qvm-open-in-vm`/`qvm-open-in-dvm`.